python/pyspark/ml/feature.py

```diff
@@ -465,26 +522,26 @@ class CountVectorizer(JavaEstimator, HasInputCol, HasOutputCol, JavaMLReadable,
         " Default False", typeConverter=TypeConverters.toBoolean)
 
     @keyword_only
-    def __init__(self, minTF=1.0, minDF=1.0, vocabSize=1 << 18, binary=False, inputCol=None,
-                 outputCol=None):
+    def __init__(self, minTF=1.0, minDF=1.0, maxDF=2 ** 63 - 1, vocabSize=1 << 18, binary=False,
```

Comment:
I'm not crazy about hardcoding a value here since in Scala it is `Long.MaxValue`, but I'm not sure there is another way.

Reply:
Thank you very much for the comments. Will make changes.